ups/ts_MTAOS.table

```diff
@@ -3,6 +3,7 @@
 # - Common third-party packages can be assumed to be recursively included by
 #   the "sconsUtils" package.
 setupRequired(sconsUtils)
+setupRequired(lsst_distrib)
```

Comment:
Base on the code, it looks like you only need to setup the `lsst.daf` to use the `butler`. But @jbkalmbach may need to update the table file of `ts_wep` to add the `setupRequired` of `ctrl_mpexec` based on the `Jenkinsfile` in `ts_wep`.